plugins/terminal.koplugin/main.lua

```diff
@@ -66,12 +68,32 @@ function Terminal:execute()
         table.insert(entries, _("Execution canceled."))
     end
     UIManager:close(wait_msg)
-    UIManager:show(TextViewer:new{
+    local viewer
+    viewer = TextViewer:new {
         title = _("Command output"),
         text = table.concat(entries, "\n"),
         justified = false,
         text_face = Font:getFace("smallinfont"),
-    })
+        -- support re-invoking the terminal with a button:
+        buttons_table = {
+            {
+                {
+                    text = 'New command(s)...',
```

Comment:
I'd like something better, but no idea what. Re-run ? Edit ?
Also, our prefered quotes for strings are "...", not '...'

Reply:
I think edit would be welcome but that's not what this does?

In this context, probably just `Run` would suffice.

Reply:
I've now updated this to "Back". I have almost no experience with PR's: do you see my edits automatically, or do I have to do a PR again?

Reply:
@smartscripts-nl You just commit and push to your remote branch, PR will update automatically ;-)